programs/clearing_house/src/lib.rs

```diff
@@ -684,7 +645,10 @@ pub mod clearing_house {
         )?;
 
         validate!(
-            spot_market.status == MarketStatus::Initialized,
+            !matches!(
+                spot_market.status,
+                MarketStatus::ReduceOnly | MarketStatus::Settlement | MarketStatus::Delisted
```

Comment:
for reduce only, user could be paying off borrow with this function?

Reply:
yea good point.. would this catch?

```
if spot_position.balance_type == SpotBalanceType::Deposit && spot_position.balance > 0 {
            validate!(
                !matches!(
                    spot_market.status,
                    MarketStatus::ReduceOnly | MarketStatus::Settlement | MarketStatus::Delisted
                ),
                ErrorCode::DefaultError,
                "spot_market in reduce only mode",
            )?;
        }
```